vtex/faststore
# Feat/add checkout dev env

## What's the purpose of this pull request?

To ensure smooth functionality across different environments, such as localhost and development setups, it's crucial to include the necessary parameters for checkout. 
This practice maintains the integrity of the store-to-checkout flow. While in the final environment structure, domain and subdomain configurations enable seamless checkout using the cookie implementation, it becomes essential to explicitly pass parameters during testing to uphold the continuity of the process.

## How it works?

Add ?orderFormId param when using localhost or vtex.app env.

## How to test it?

Go to the checkout using vtex.app env or localhost and check the parameter passing at the URL.

### Starters Deploy Preview
https://github.com/vtex-sites/starter.store/pull/382

https://github.com/vtex/faststore/assets/67066494/666205f8-6e8a-49ca-bab4-25840d47517e
